triplets/x64-osx.cmake

```diff
@@ -5,3 +5,7 @@ set(VCPKG_LIBRARY_LINKAGE static)
 set(VCPKG_CMAKE_SYSTEM_NAME Darwin)
 set(VCPKG_OSX_ARCHITECTURES x86_64)
 
+set(VCPKG_OSX_DEPLOYMENT_TARGET 10.12)
+set(VCPKG_C_FLAGS -mmacosx-version-min=10.12)
+set(VCPKG_CXX_FLAGS -mmacosx-version-min=10.12)
```

Comment:
I would prefer to keep this in the GH Actions workflow file than hack vcpkg's source code. Alternatively we may make our own custom triplet.

Reply:
Yes, Ok.